ORGANISATIONS.yaml

```
@@ -337,6 +344,13 @@ societyprotectionundergroundnetworks:
     url: "https://www.spun.earth"
     avatar: "/training-material/shared/images/spun-logo.png"
 
+spoc:
+    short_name: 🖖🏾SPOC
+    name: 🖖🏾Single-cell & sPatial Omics Community of practice
+    bio: We are a mix of users, developers, and trainers providing feedback and energy to improve and expand the Galaxy single cell and spatial omics capabilities.
+    url: https://galaxyproject.org/community/sig/singlecell/
+    avatar: "shared/images/logos/spoc_hand.png"
```

```suggestion
    avatar: "/training-material/shared/images/logos/spoc_hand.png"
```

terribly inconsistent of us, but these need the /training-matarial/ prefix it appears